### PR TITLE
ACA-Py 0.10.1 update

### PR DESCRIPTION
**Pull request by loneil**
Update ACA-Py docker image. Poetry update plugins. Change validator imports from messaging/valid.py to account for breaking change.

https://github.com/bcgov/traction/issues/798

### CODE REVIEW

**Comment by loneil** [comment]
Issue locally... looking into

```
scripts-traction-agent-1                | Shutting down
scripts-traction-agent-1                | 2023-09-06 04:45:02,887 aries_cloudagent.commands.start ERROR Exception during startup:
scripts-traction-agent-1                | Traceback (most recent call last):
scripts-traction-agent-1                |   File "/home/aries/.venv/lib/python3.9/site-packages/aries_cloudagent/commands/start.py", line 72, in init
scripts-traction-agent-1                |     await startup
scripts-traction-agent-1                |   File "/home/aries/.venv/lib/python3.9/site-packages/aries_cloudagent/commands/start.py", line 28, in start_app
scripts-traction-agent-1                |     await conductor.setup()
scripts-traction-agent-1                |   File "/home/aries/.venv/lib/python3.9/site-packages/aries_cloudagent/core/conductor.py", line 122, in setup
scripts-traction-agent-1                |     context = await self.context_builder.build_context()
scripts-traction-agent-1                |   File "/home/aries/.venv/lib/python3.9/site-packages/aries_cloudagent/config/default_context.py", line 65, in build_context
scripts-traction-agent-1                |     await self.load_plugins(context)
scripts-traction-agent-1                |   File "/home/aries/.venv/lib/python3.9/site-packages/aries_cloudagent/config/default_context.py", line 144, in load_plugins
scripts-traction-agent-1                |     plugin_registry.register_plugin(plugin_path)
scripts-traction-agent-1                |   File "/home/aries/.venv/lib/python3.9/site-packages/aries_cloudagent/core/plugin_registry.py", line 128, in register_plugin
scripts-traction-agent-1                |     mod = ClassLoader.load_module(module_name)
scripts-traction-agent-1                |   File "/home/aries/.venv/lib/python3.9/site-packages/aries_cloudagent/utils/classloader.py", line 70, in load_module
scripts-traction-agent-1                |     return import_module(mod_path, package)
scripts-traction-agent-1                |   File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
scripts-traction-agent-1                |     return _bootstrap._gcd_import(name[level:], package, level)
scripts-traction-agent-1                |   File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
scripts-traction-agent-1                |   File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
scripts-traction-agent-1                |   File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
scripts-traction-agent-1                |   File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
scripts-traction-agent-1                |   File "<frozen importlib._bootstrap_external>", line 850, in exec_module
scripts-traction-agent-1                |   File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
scripts-traction-agent-1                |   File "/home/aries/traction_plugins/traction_innkeeper/v1_0/__init__.py", line 11, in <module>
scripts-traction-agent-1                |     from . import (
scripts-traction-agent-1                |   File "/home/aries/traction_plugins/traction_innkeeper/v1_0/schema_storage/__init__.py", line 8, in <module>
scripts-traction-agent-1                |     from .schema_storage_service import SchemaStorageService, subscribe
scripts-traction-agent-1                |   File "/home/aries/traction_plugins/traction_innkeeper/v1_0/schema_storage/schema_storage_service.py", line 16, in <module>
scripts-traction-agent-1                |     from .models import SchemaStorageRecord
scripts-traction-agent-1                |   File "/home/aries/traction_plugins/traction_innkeeper/v1_0/schema_storage/models.py", line 4, in <module>
scripts-traction-agent-1                |     from aries_cloudagent.messaging.valid import INDY_SCHEMA_ID
scripts-traction-agent-1                | ImportError: cannot import name 'INDY_SCHEMA_ID' from 'aries_cloudagent.messaging.valid' (/home/aries/.venv/lib/python3.9/site-packages/aries_cloudagent/messaging/valid.py)
```

**Comment by loneil** [comment]
Change here to `messaging/valid.py` removed a few validators we imported to use

https://github.com/hyperledger/aries-cloudagent-python/pull/2398/files#diff-d108b51a8a195f4e1a8dbfd6fd071f5d8b36b6623d0c8ff012ef853936397b6c

**Comment by loneil** [comment]
Need to do a bit more regression (tested out and found issues with schema/cred record validation I've fixed) but can take a look at code changes if anything sticks out.

File: plugins/docker/Dockerfile [hunk 1]
```diff
@@ -27,7 +27,7 @@ RUN poetry install --no-dev
 RUN ln -s $(poetry env info -p)/lib/python3.6/site-packages site-packages
 
 
-FROM ghcr.io/hyperledger/aries-cloudagent-python:py3.9-0.9.0
+FROM ghcr.io/hyperledger/aries-cloudagent-python:py3.9-0.10.1
```
Comment (esune) [inline plugins/docker/Dockerfile:30]: Unrelated to the PR objective, but noticing this now: base and runtime steps use the same base image, we should be able to use one stage only (I am not sure this is actually providing us with a benefit).

**Review by esune — APPROVED** [review]
Looks good to me 👍🏻

**Review by usingtechnology — APPROVED** [review]

File: plugins/docker/Dockerfile (hunk 1 above)
Comment (esune) [reply plugins/docker/Dockerfile:30]: Ah, I see - it's to optimize the final image size. We could just uninstall the poetry package after we've completed the build, or leave it as-is. Definitely not a big issue and not something to hold-up this PR.

**Pull request merged**